pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/model/ras/Ga4ghPassportV1.java

```diff
@@ -0,0 +1,109 @@
+package edu.harvard.hms.dbmi.avillach.auth.model.ras;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class Ga4ghPassportV1 {
```

Comment:
It might be a pain, but do we want to name these what they actually stand for and json map them individually? At the very least we should add documentation what they mean